src/DynamoCoreWpf/Views/Core/DynamoView.xaml

```diff
@@ -1600,22 +1601,14 @@
                             </Border>
                         </ControlTemplate>
                     </Button.Resources>
-                    <Grid Mouse.MouseEnter="Button_MouseEnter"
-                          Mouse.MouseLeave="Button_MouseLeave">
+                    <Grid Mouse.MouseEnter="ExtensionHandle_MouseEnter"
+                          Mouse.MouseLeave="ExtensionHandle_MouseLeave">
                         <Grid.ColumnDefinitions>
                             <ColumnDefinition Width="*" />
                             <ColumnDefinition Width="Auto" />
                         </Grid.ColumnDefinitions>
                         <StackPanel Orientation="Horizontal" 
                                     HorizontalAlignment="Center">
-                            <TextBlock Grid.Column="0"
-                                   Name="ExtensionSidebarText"
-                                   VerticalAlignment="Center"
-                                   Foreground="#aaaaaa"
-                                   FontWeight="Normal"
-                                   Margin="5,0,0,5"
-                                   Text="{x:Static p:Resources.ExtensionsViewTitle}">
-                            </TextBlock>
```

Comment:
Curious why did we only move this textblock for the right side but keep the change for left?

Reply:
Without getting into much detail, the main difference is that the centre of the rotation is different for each side, which in turn requires to do things differently in order to show the handles as we expect them.